server/notify.py

```diff
@@ -0,0 +1,87 @@
+from web3 import Web3
+import discord
+import asyncio
+import requests
+
+# GLOBALS
+# for web3 integration
+ETHEREUM_NODE_URL = 'YOUR_ETHEREUM_NODE_URL' 
+CONTRACT_ADDRESS = 'YOUR_CONTRACT_ADDRESS'
+CONTRACT_ABI = []

```

Comment:
Error here : 
Traceback (most recent call last):
  File "/Users/mehdi/hackathon/bdmh-cross-chain-governance/server/notify.py", line 64, in <module>
    "anonymous": false,